tests/test_routing.py

```diff
@@ -858,7 +858,7 @@ def test_distance_to_channel_mfd(self):
             [2400, 17984, 17984, 17984, 17984, 17984, 17984, 17984, 17984,
              17984, 26880],
             [2400, 17984, 17984, 17984, 17984, 17984, 17984, 17984, 17984,
-             17984, 26880]], dtype=numpy.int32)
+             1178599424, 26880]], dtype=numpy.int32)
```

Comment:
this is changing an edge pixel to flow off the array?

Reply:
yes! in fact I was using your 1178599424 as the example you dug up in your earlier question.